src/main/java/net/datafaker/providers/base/Barcode.java

```diff
@@ -67,23 +67,24 @@ private long ean(int length) {
             even = odd;
             odd = tmp;
         }
-        int var = 0;
-
-        if (length == 13) {
-            var = odd + even + (even << 1);
-        } else if (length == 8 || length == 14 || length == 12) {
-            var = odd + even + (odd << 1);
-        }
-
-        int rounded = roundToHighestMultiplyOfTen((var));
-        int checkDigit = rounded - var;
+        int output = calculateOutput(length, odd, even);
```

Comment:
frankly speaking i'm not sure that `output` is a good name here... Did you check how this values called in normal `ean` generation process?

Reply:
> frankly speaking i'm not sure that `output` is a good name here... Did you check how this values called in normal `ean` generation process?

It was initially named as var, since it was suggested in the issue to rename var since there is a similar keyword in Java now.